client/router_create.go

```diff
@@ -1081,6 +1087,63 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 	return van
 }
 
+func (cli *VanClient) GetRouterHostAliasesSpecFromTokens(ctx context.Context) ([]corev1.HostAlias, error) {
+	hostAliasesMap := make(map[string]map[string]bool)
+	secrets, err := cli.KubeClient.CoreV1().Secrets(cli.GetNamespace()).List(ctx, metav1.ListOptions{LabelSelector: types.SkupperTypeQualifier + "=" + types.TypeToken})
+	if err != nil {
+		return nil, fmt.Errorf("Failed to retrieve connection token: %w", err)
+	}
+	if len(secrets.Items) == 0 {
+		return nil, fmt.Errorf("Connection token not found")
```

Comment:
This doesn't seem like an error, just the absence of any aliases at present.

Reply:
Good point. Will change to `return nil, nil`